specification/cognitiveservices/data-plane/FormRecognizer/stable/v1.0/FormRecognizer.json

```diff
@@ -22,7 +22,7 @@
         "useSchemePrefix": false,
         "parameters": [
             {
-                "$ref": "#/parameters/Endpoint"
+                "$ref": "../../../Common/Parameters.json#/parameters/GlobalEndpoint"
```

Comment:
Please use local Endpoint configuration.
The common file will be removed

Reply:
removed Endpoint parameter.